src/FSLibrary/FSLibrary.fsproj

```diff
@@ -66,6 +67,7 @@
     <Compile Include="MissionProtocolUpgradeWithLoad.fs" />
     <Compile Include="MissionDatabaseInplaceUpgrade.fs" />
     <Compile Include="MissionAcceptanceUnitTests.fs" />
+    <Compile Include="MissionMaxTPSMixed.fs" />
```

Comment:
To be consistent with naming, should we rename `MissionSimulatePubnetTier1Perf` to `MissionMaxTPSClassic`? (note that this would be a breaking change and require updates to CI)

Reply:
Done!